gradle.properties

```diff
@@ -1,3 +1,3 @@
 publishedPluginVersion=1.18.4
-pluginVersion=1.18.5-pre16
+pluginVersion=1.18.5-pre17
```

Comment:
hmmm ... current version is 1.19.1-pre6 :D
btw please add license headers in new files

Reply:
still missing license header in test file